test/Microsoft.IdentityModel.TestUtils/KeyingMaterial.cs

```diff
@@ -71,9 +85,15 @@ static public class KeyingMaterial
         public static SigningCredentials X509SigningCreds_SelfSigned2048_SHA512 = new SigningCredentials(X509SecurityKeySelfSigned2048_SHA512, SecurityAlgorithms.RsaSha512);
 
         // all asymmetric material has private key unless public is included in variable name
-        public static SecureString CertPassword = ConvertToSecureString("abcd");
+        public static string CertPasswordString = "abcd";
+        public static SecureString CertPassword = ConvertToSecureString(CertPasswordString);
```

Comment:
Why was this change and the ones above made?

Reply:
Ah they're vestigial to before we decided to pragma warning disable since the constructor didn't have an overload. X509CertificateLoader.LoadPcks12() has similar args but with a string instead of secure string password arg. But we won't be using that - so reverting these changes. Thank you!

Reply:
Reverted these changes.